api.planx.uk/inviteToPay/sendPaymentEmail.ts

```diff
@@ -0,0 +1,83 @@
+import { gql } from "graphql-request";
+import { Template, calculateExpiryDate, convertSlugToName, getClientForTemplate, sendEmail } from "../saveAndReturn/utils";
+import { InviteToPayNotifyConfig } from "../types";
+
+const sendSinglePaymentEmail = async (
+  template: Template,
+  paymentRequestId: string,
+) => {
+  try {
+    const payment = await validatePaymentRequest(paymentRequestId, template);
+    if (!payment || Boolean(payment?.paid_at)) throw Error(`Invalid payment request: ${paymentRequestId}`);
+
+    const config: InviteToPayNotifyConfig = {
+      personalisation: {
+        id: payment.session_id,
+        serviceName: convertSlugToName(payment.session.flow.slug),
+        emailReplyToId: payment.session.flow.team.notify_personalisation.emailReplyToId,
+        helpEmail: payment.session.flow.team.notify_personalisation.helpEmail,
+        helpOpeningHours: payment.session.flow.team.notify_personalisation.helpOpeningHours,
+        helpPhone: payment.session.flow.team.notify_personalisation.helpPhone,
+        paymentRequestId: paymentRequestId,
+        payeeEmail: payment.payee_email,
+        payeeName: payment.payee_name,
+        agentName: `{TODO}`,
+        paymentLink: `https://${payment.session.flow.team.domain}/${payment.session.flow.slug}/pay?paymentRequestId=${paymentRequestId}`,
+        fee: (payment.session_preview_data.fee).toLocaleString("en-GB", { style: "currency", currency: "GBP" }),
+        projectType: payment.session_preview_data?.["proposal.projectType"],
+        address: payment.session_preview_data?._address.title,
+        expiryDate: calculateExpiryDate(payment.created_at),
+      }
```

Comment:
this `personalisation` type & mapping is my first draft / best guess - I fully anticipate we'll keep refining this as we refine the `payment_requests` schema and the content team take a pass at the Notify templates themselves. 

it should be simple to pick up in small, follow-up PRs.